gradle-plugin/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Mon May 04 12:41:37 CEST 2015
+#Fri Sep 25 13:40:00 CEST 2015
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-2.4-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-2.0-all.zip
```

Comment:
Yeah. I can put the version in a properties file and reference that across the projects. Also useful to centralize SDK and NDK versions